homework/english/maksim_shelkovich/lecture1/task1.md

```diff
@@ -0,0 +1,3 @@
+# Homework 1 task 1
+
+Hello world! this is my homework.
```

Comment:
This is not a homework. Plese, do HW!

- [ ]  add at least 1 task

Reply:
ok